### PR TITLE
support new release API

### PR DESCRIPTION
**Pull request by michaelglass**
~~ ready for review but blocked on the new API getting released~~

now that we're not trying to align run id and install id, I believe that we don't need to make install-id customisable anymore on github.

Until the new API is released, CI will be red.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -18,8 +18,3 @@ inputs:
       ABQ Release Channel.
     required: false
     default: v1
-  run-id:
-    description: |
-      Unique run attempt ID. One will be generated from the GitHub
-      RUN_ID and RUN_ATTEMPT if available.
-    required: false
```
Comment (dan-manges) [inline action.yml:25]: Do GitHub Actions support passing invalid inputs? If somebody is passing `run-id`, will their integration continue to work?

Comment (michaelglass) [reply action.yml:25]: If I recall correctly, it's ignored but I'll double check.

Comment (michaelglass) [reply action.yml:25]: tried it [here](https://github.com/rwx-research/setup-abq/commit/ce5be0d8b606a82b2c5b316383322aa01ee89ec6)

```
Warning: Unexpected input(s) 'made-up-config-that-does-not-exist', valid inputs are ['install_url', 'install_options', 'nix_path', 'extra_nix_config']
```
then just ignored it

Comment (michaelglass) [reply action.yml:25]: pulled in github's deprecationMessage to more explicitly nudge users away from using `run-id`

**Review by doxavore — APPROVED** [review]

**Pull request merged**